src/features/academic/components/shared/Accordion/CourseList/CourseList.module.scss

```
@@ -37,6 +37,13 @@
   color: color.$gray-100;
 }
 
+// 선택교양(선교) 과목의 세부 영역 표시 ("{liberalAreaCode}영역"). 학기 텍스트와 동일 톤.
+.area {
+  @include typography.body-sm;
+  @include typography.font-weight-medium;
+  color: color.$gray-100;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)